### ISSUE TITLE
Allow multiple google accounts per Subscriber

### ISSUE THREAD

**Issue by radishmouse** [thunderbird/appointment#123]
**Is your feature request related to a problem? Please describe.**

Currently, each `Subscriber` is limited to a single Google account, since we store the `google_tkn` in the `Subscriber` table.

However, it's likely that users may have multiple Google accounts that they want to connect to.

**Describe the solution you'd like**

- Move the `google_*` fields to a new `Accounts` table
- Link the `Accounts` table to the `Subscriber` table
- Link the `Calendar` table to the `Accounts` table

(If it makes sense, we could also remove the `owner` field from the `Calendar` table, and derive that knowledge from the linked `Accounts` table)

**Describe alternatives you've considered**

If multiple Google account support isn't on the roadmap, feel free to disregard this ticket!

**Comment by umgpy** [comment]
Any update on this issue? Really an important feature given most have work and personal google accounts.

**Comment by malini** [comment]
@umgpy yes, it's targeted for release by the end of May, however, we can prioritize this issue to see if we can get it in earlier.

**Comment by umgpy** [comment]
Thanks @malini!
Would be happy to test and debug whenever something is ready.

**Comment by davinotdavid** [comment]
@radishmouse as the PR has been merged in, can we close this issue? Happy to continue working on it if there's anything else that you see as a requirement of course 🙇

**Comment by radishmouse** [comment]
Closed by #1078
